h/templates/deform/select.jinja2

```diff
@@ -0,0 +1,14 @@
+<select
+{% include "includes/common_attrs.jinja2" %}
+>
+{% for value, description in field.widget.values %}
+ <option
+        {% if value == cstruct %}
+        selected="selected"
+        {% endif %}
+        {% if field.widget.css_class %}
+        class="{{ field.widget.css_class }}"
+        {% endif %}
+        value="{{ value }}">{{ description }}</option>
+{% endfor %}
+</select>
```

Comment:
You might be aware of this but the group type select doesn't seem to work right:

![peek 2018-03-02 13-07](https://user-images.githubusercontent.com/22498/36900338-d9eb5946-1e1a-11e8-8384-ecc056a196c2.gif)

Reply:
@seanh D'oh! I think we're dealing with a browser issue here, as it works fine in my Chrome 64 on Mac OS. What browser/system are you on? Are there any other symptoms (e.g. errors in console, though there really shouldn't be as there is no JS involved here that I know of)...?

Reply:
Hmm, I'm seeing this in both Chrome and Firefox (on linux). I wonder if there's something wrong on my end though. Will investigate...

Reply:
No errors in console or terminal. Hmmmm

Reply:
@seanh Might try inspecting (CSS) for the element and unchecking some of the styles to see if that helps? I added a couple of new style rules for the select element there.

Reply:
Eh, I can't seem to get rid of it. Confounded

Reply:
![dropdown](https://user-images.githubusercontent.com/439947/36903661-60647f94-1dfc-11e8-8b01-24afcc8d21fa.gif)

For reference: Chrome 64 on Mac OS